api/chat_gpt_utils.py

```diff
@@ -0,0 +1,60 @@
+"""Script for Open AI functions."""
+
+from dotenv import load_dotenv
+from openai import OpenAI
+
+GPT_3_MODEL = 'gpt-3.5-turbo-1106'
+GPT_4_MODEL = 'gpt-4-1106-preview'
+
+
+load_dotenv()
+
+
+def read_html_file(file_path: str) -> str:
+    """Reads in HTML file."""
+
+    with open(file_path, 'r', encoding='utf-8') as f:
+        html_content = f.read()
+    return html_content
```

Comment:
Could be rewritten as:
```suggestion
    with open(file_path, 'r', encoding='utf-8') as f:
        return f.read()
    
```